frontend/src/metabase/collections/components/CollectionSidebar/Bookmarks/getIcon.ts

```diff
@@ -0,0 +1,18 @@
+import { BookmarkableEntities } from "metabase-types/api";
```

Comment:
nip: noticed `BookmarkableEntities` only includes `cards` and `collections`, although dashboards can be bookmarked too

Reply:
@kulyk Well caught. I am adding this in a follow-up PR in the works, one about Official Collections. Is it ok for you?

Reply:
Namely, here: https://github.com/metabase/metabase/pull/21240